scripts/config.pl

```diff
@@ -170,6 +172,9 @@
 if ($action eq "realfull") {
     $exclude_re = qr/^$/;
     $no_exclude_re = qr/./;
+} elsif ($action eq "docfull") {
+    $exclude_re = qr/_ALT\s*$/;
+    $no_exclude_re = qr/(?:PLATFORM|ECP)_[_A-Z0-9]+_ALT\s*$/;
```

Comment:
This also matches double (or more) `_`. Consider removing all `_` from outside the `[]`.

Reply:
This commit says it doesn't include ECC functions in doxygen output yet, but it perhaps is partially making the attempt by having `ECP` in this no exclude regex. Should the `|ECP` addition be in the ECC function no exclude commit instead?

Reply:
I think the structure of the commits makes logical sense as they are. I don't see a need to restructure them.